cli/defenseclaw/commands/cmd_upgrade.py

```
@@ -1960,53 +2018,76 @@ def _maybe_delegate_public_upgrade(argv: list[str]) -> None:
-                        "https://token.actions.githubusercontent.com",
-                        paths["checksums.txt"],
-                    ],
-                    capture_output=True,
-                    text=True,
-                    timeout=30,
-                    check=False,
-                )
-            if verified.returncode != 0:
-                raise OSError("resolver checksum signature did not match the DefenseClaw release workflow")
+                with _isolated_cosign_environment(trusted_environment) as cosign_environment:
+                    channel_path, channel_bundle_path = _download_authenticated_channel_pair(
```

Ruff (0.15.21)

[warning] Unpacked variable `channel_bundle_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)